conans/client/installer.py

```diff
@@ -227,6 +225,14 @@ def build_package(self, node, keep_build, recorder, remotes):
             return node.pref
 
 
+def remove_folder_raising(folder):
+    try:
+        rmdir(folder)
+    except OSError as e:
+        raise ConanException("%s\n\nCouldn't remove folder, might be busy or open\n"
+                             "Close any app using it, and retry" % str(e))
```

Comment:
Does the exception message include the folder itself? I would want to let the user know which one is the folder that might be in use by other app...

Reply:
Sure, the OSError exception includes the path of the folder